beam-core/beam-core.cabal

```diff
@@ -2,7 +2,7 @@
 -- see http://haskell.org/cabal/users-guide/
 
 name:                beam-core
-version:             0.8.1.0
+version:             0.8.2.0
```

Comment:
Don't worry about updating the version number, we'll handle that before releases.